SJM and All runtimes latest release changes (#21345)
newrelic/docs-website: src/content/docs/release-notes/synthetics-release-notes/job-manager-release-notes/job-manager-release-451.mdx

---
subject: Job Manager
releaseDate: '2025-07-28'
version: '451'
---
### Fixes
* sjm vulnerability fixes

> * processor queue migration  changes

Reviewer: @schinthapatla Please provide more details what we have fixed here? The information is incomplete here.

> * null pointer logging

Reviewer: @schinthapatla  Please provide more details what we have fixed here? The information is incomplete here. Thanks

> * Fixed common-lang3 java related vulnerability and  Ubuntu vulnerabilities to address security vulnerabilities in Synthetics Job Manager.

Reviewer: ```suggestion
* Fixed vulnerabilities related to `common-lang3` Java and Ubuntu to address security issues in the SJM.
```

> * processor queue migration changes - Changes related to the migration of our internal service architecture.

Reviewer: ```suggestion
* Implemented processor queue migration changes related to the migration of our internal service architecture.
```

> * Currently customer is getting null cause in SJM logs , hence we are logging complete exception to ensure that the customer has some context.

Reviewer: ```suggestion
* Fixed an issue to include full exception details to provide better context for troubleshooting in SJM logs.
```
